README.md

```diff
@@ -67,6 +67,10 @@ Once you have a decent set of coefficients and biases computed, paste them into
 
 Thanks to Treora for his excellent freeze-dry library!
 
+## Development
+
+To work on FathomFox itself, do the steps under "Get FathomFox Running" above.
```

Comment:
Optional, but you could add a relative link to the "Get FathomFox Running" section of the README from the "Development" section. Here's an [example](https://github.com/mozilla/price-tracker/blob/master/docs/METRICS.md) that uses an in-page, relative URL for the "Common Extra Keys" section.

Reply:
Done!